omod/src/main/webapp/resources/scripts/emr.js

```diff
@@ -87,6 +87,37 @@ var emr = (function($) {
             }
         },
 
+        serverValidationErrorMessage: function(response) {
+    		var errorMessage = this.message("uicommons.generalError.message", "An error has occurred");
+        	if (response.data != null && response.data.error != null) {
+        		if (response.data.error.message != null) {
+	        		errorMessage = response.data.error.message;
+	            	if (response.data.error.globalErrors != null) {
+	                	response.data.error.globalErrors.forEach(function (val, idx){
```

Comment:
forEach works for array. Assuming the array is not null.
This code could not be tested. I mangled the fields for saveRole as much as I could but the REST-API always returned an internal server error, (500) which allowed only the code which outputs the errorMessage to be tested. I could not get the REST-API to return a validation message as we discussed.